assets/schema_input.json

```diff
@@ -12,76 +12,76 @@
                 "pattern": "^\\S+$",
                 "meta": ["id"],
                 "unique": true,
-                "errorMessage": "Sample name must be provided and cannot contain spaces"
+                "errorMessage": "Sample name must be provided and cannot contain spaces."
             },
             "mlst_alleles": {
                 "type": "string",
                 "format": "file-path",
                 "pattern": "^\\S+\\.mlst(\\.subtyping)?\\.json(\\.gz)?$",
-                "errorMessage": "MLST JSON file from locidex report, cannot contain spaces and must have the extension: '.mlst.json' or '.mlst.json.gz'"
+                "errorMessage": "MLST JSON file from locidex report, cannot contain spaces and must have the extension: '.mlst.json' or '.mlst.json.gz'."
```

Comment:
The extension could also be `.mlst.subtyping.json` or `.mlst.subtyping.json.gz`. Could you mention this?

Reply:
Updated in 38f58e6ea1b7360d14a346b633bbf19d35c5818c